### PR TITLE
add badges to README

### PR DESCRIPTION
**Pull request by biantris**
closes: https://github.com/FuelLabs/fuels-ts/issues/139
- adding fuels-ts related badges
image

### CODE REVIEW

**Review by QuinnLee — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 **fuels-ts** is a library for interacting with **Fuel v2**.
 
+[![build](https://github.com/FuelLabs/fuels-ts/actions/workflows/ci.yaml/badge.svg)](https://github.com/FuelLabs/fuels-ts/actions/workflows/ci.yaml)
+[![npm](https://img.shields.io/npm/v/@fuel-ts/contract)](https://www.npmjs.com/package/@fuel-ts/contract)
```
Comment (AlicanC) [inline README.md:6]: I think this should point to `fuels` and not `@fuel-ts/contract`.

**Review by AlicanC — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (biantris) [reply README.md:6]: Ok I will upgrade to this package https://www.npmjs.com/package/fuels

**Review by luizstacio — APPROVED** [review]

**Comment by adlerjohn** [comment]
Poking @AlicanC for review!

**Comment by biantris** [comment]
Hi @AlicanC could you take a look here, thanks!

**Review by AlicanC — APPROVED** [review]
LGTM!

**Pull request merged**